### ISSUE TITLE
"The server must have JEI installed" error

### ISSUE THREAD

**Issue by davetassoni** [mezz/JustEnoughItems#1907]
I am running a Forge dedicated server, with jei_1.12.2-4.15.0.293.jar on both the server and the jar. I get this issue when hovering over the + symbol in the crafting menu for an item. I cannot determine how to fix the issue. Logs posted below:

[Server Log](https://pastebin.com/URkcpX3R)

[Client Log](https://pastebin.com/QeyRNaR1)

**Comment by 1legotrain** [comment]
What did you do to fix this? I'm having the same issue.

**Comment by davetassoni** [comment]
You have to make sure that you are running the server from the forge jar and not the Minecraft server jar. Also be sure that the mods are on the server

**Comment by madisonlove21** [comment]
I had the same problem and I finally figured out how to fix it! It seemed no matter what I did JEI wouldn't work (even if it was the ONLY mod). So on the Shockbyte multicraft server go to "Files" then click "Modpack Installer". On the "Template" line click the drop down list and select your version of forge as the modpack installer. Be sure to click Apply and restart the server. Hope this helps!
